test/routes/index-test.js

```diff
@@ -0,0 +1,21 @@
+const { expect } = require('chai')
+const got = require('got');
+
+const { server } = require('../../src/index') 
+
+describe('Index page', () => {
+    before(async () => {
+        await server.start();
+        // console.log(`Server started on ${server.host}:${server.port}`);
```

Comment:
It's better to not use comments for disabling code. Also, debug information can be very useful. I think we can remove `//` and move these messages into the server code. Don't worry, we will implement a more elegant debug reporting solution as a next task 😄